templates/snyk-ecr-auto.template.yaml

```
@@ -9,7 +9,7 @@ Outputs:
     Description: Amazon Resource Name (ARN) role for your Snyk integration.
     Value: !GetAtt 'SnykECRIntegrationRole.Arn'
   Version:
-    Description: Snyk ECR integration version
+    Description: Snyk integration version.
```

Change this to Snyk Amazon ECR integration version (similar to Snyk AWS Lambda integration version in the snyk-lambda template)
```suggestion
    Description: Snyk Amazon ECR integration version
```